CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.22] - 2020-06-08
```

Comment:
This line needs to be removed as it implies it's released. You only add the date when it's been merged to master

Reply:
ok cool